hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -830,7 +832,7 @@ message LookupKeyResponse {
 
 message RenameKeyRequest{
     required KeyArgs keyArgs = 1;
```

Comment:
Can we have a list of KeyArgs here?

Reply:
Agree. Use list of KeyArgs instead of add a new keyNameList field.

Reply:
Agree. Use list of KeyArgs instead of add a new keyNameList field.

Reply:
Agree. Use list of KeyArgs instead of add a new keyNameList field.

Reply:
Agree. Use list of KeyArgs instead of add a new keyNameList field.

Reply:
Very good advice. Instead of modifying KeyArgs, I'll try to implement delete with a List that contains KeyArgs.